API.md

```diff
@@ -119,9 +133,73 @@ Every error will contain a root element named `error` with an error message.
     }]
 }
 ```
+**Response**
+```json
+{
+    "data": "ok"
+}
+```
+
+## Administrator
+### Request structure
+Every query will need to contain the key `administratorPassword` with the right password.
+```json
+{
+    "administratorPassword": "string",
+    ...
+}
+```
+
+### Launch competition
+Note that putting running to `false` won't have any effect.
+
+`PATCH /admin/competition`
+
+**Request**
+```json
+{
+    "running": "boolean"
+}
+```
+**Response**
+```json
+{
+    "data": "ok"
+}
+```
+
+### Set ranked game duration
+This will set the duration of the next ranked game.
+
+`PATCH /admin/ranked`
 
+**Request**
+```json
+{
+    "duration": "int"
+}
+```
+**Response**
 ```json
 {
     "data": "ok"
 }
 ```
+
+### Create player
+`POST /admin/player`
+
+**Request**
+```json
+{
+    "playerName": "string"
+}
+```
+**Response**
+```json
+{
+    "data": {
+        "playerSecret": "string"
```

Comment:
Actually, we'd need a `GET` that returns the `playerId`s and `playerSecret`s, it's not actually needed for the creation of players.